### PR TITLE
Fixes notification setting

### PR DESCRIPTION
**Pull request by Adebesin-Cell**
fixes https://github.com/EveripediaNetwork/issues/issues/809

# Changes
- refactors ActivityCard to take only props it needs
- creates a new rtk query for wiki info sufficient to render ActivityCards
- creates rtk queries for getSubscriptions, addSubscriptions, removeSubscriptions
- Implements wiki notification subscription feature in user settings page
- Removes existing checkbox logic on notifications

# Links
- https://iq-wiki-git-fixes-nofification-setting-prediqt.vercel.app/account/settings?tab=notifications

# Screenshots
screely-1669049560810

### CODE REVIEW

**Comment by Adebesin-Cell** [comment]
@Simplypheyie96 I think we might have to use a + icon for adding and - icon for removal on mobile.

**Comment by Simplypheyie96** [comment]
> @Simplypheyie96 I think we might have to use a + icon for adding and - icon for removal on mobile.

That is what i said 😐

**Comment by Simplypheyie96** [comment]
I have updated it with the new buttons for mobile @Adebesin-Cell

**Comment by Adebesin-Cell** [comment]
Thanks @Simplypheyie96, I'll work on it now.

**Review by Royal-lobster — APPROVED** [review]

**Review by Simplypheyie96 — APPROVED** [review]
it looks good now.
But the addition of wikis from recommendations feels unnatural. I guess it's because this is not yet merged right?

**Comment by Royal-lobster** [comment]
> it looks good now. But the addition of wikis from recommendations feels unnatural. I guess it's because this is not yet merged right?

@Simplypheyie96 do you mean the animations ?

File: src/theme/semantic-tokens.ts [hunk 1]
```diff
@@ -201,6 +213,14 @@ export const semanticTokens: SemanticTokens = {
...
       _dark: 'whiteAlpha.50',
     },
+    NotificationRemoveIcon: {
+      default: 'gray.800',
+      _dark: 'white',
+    },
+    NotificationAddIcon: {
+      default: 'whiteAlpha.900',
+      _dark: 'whiteAlpha.900',
+    }
```
Comment (Royal-lobster) [inline src/theme/semantic-tokens.ts:223]:
```suggestion
    },
```

**Comment by Simplypheyie96** [comment]
> > it looks good now. But the addition of wikis from recommendations feels unnatural. I guess it's because this is not yet merged right?
> 
> @Simplypheyie96 do you mean the animations ?

No i mean, when you add a wiki from recommendations, it gets added, but when you remove it, recommendations reappear. That doesn't feel natural to how recommendations work. It's not a list that goes back to its position when removed or just leaves when added.

**Comment by Royal-lobster** [comment]
> > > it looks good now. But the addition of wikis from recommendations feels unnatural. I guess it's because this is not yet merged right?
> > 
> > 
> > @Simplypheyie96 do you mean the animations ?
> 
> No i mean, when you add a wiki from recommendations, it gets added, but when you remove it, recommendations reappear. That doesn't feel natural to how recommendations work. It's not a list that goes back to its position when removed or just leaves when added.

Ohh Thanks ! we are just shuffling the recommendations when the state of subscriptions get changed, probably we can make it change less often. i will add a new commit for this.

**Comment by Simplypheyie96** [comment]
@Royal-lobster another thing is that i can't seem to check the notifications page on a tablet/a phone, so i have to change my screen size on my laptop to view. it gets stuck when i connect my wallet.I hope this would not be translated on the site after merge, right?

**Comment by Royal-lobster** [comment]
@Simplypheyie96 I patched the recommendation system, please have a look again if it makes more logical.

**Comment by Royal-lobster** [comment]
> @Royal-lobster another thing is that i can't seem to check the notifications page on a tablet/a phone, so i have to change my screen size on my laptop to view. it gets stuck when i connect my wallet.I hope this would not be translated on the site after merge, right?

Ohh, logically it should reflect the similar to the pr link, but we didn't have new changes for login flow in this pr, so it should work.

**Comment by Softdev1** [comment]
@Simplypheyie96  I think you would simply have to disconnect your devices first and connect it again because I just experience similar issue now

**Comment by Simplypheyie96** [comment]
> @Simplypheyie96  I think you would simply have to disconnect your devices first and connect it again because I just experience similar issue now 

Sorted this outnwith srujan on WhatsApp

**Review by Simplypheyie96 — APPROVED** [review]
Looks great now

**Comment by Adebesin-Cell** [comment]
Alright, @Royal-lobster are we ready to merge now?

**Pull request merged**